plugin.php

```diff
@@ -163,6 +163,9 @@
 add_action('parse_request', function(WP $pWP) use ($pDI) {
 	if (isset($pWP->query_vars['document_pdf'])) {
 		try {
+			if (empty(get_option('onoffice-settings-google-bot-index-pdf-expose'))) {
+				header('X-Robots-Tag: googlebot: noindex, nofollow');
```

Comment:
Would it be possible to integrate this into `PdfDownload` class, so that this behavior becomes testable?

Reply:
@jayay Becomes to testable, I think `PdfDocumentFetcher` class will be matching better than `PdfDownload` class, what do you think about that?